Cargo.toml

```diff
@@ -62,8 +62,8 @@ rayon = "1.10"
 regex = "1.10.5"
 serde = { version = "1.0.203", features = ["derive"] }
 serde_json = "1.0.117"
-starknet = { git = "https://github.com/xJonathanLEI/starknet-rs", rev = "d980869" }
-starknet-crypto = { git = "https://github.com/xJonathanLEI/starknet-rs", rev = "d980869" }
+starknet = { git = "https://github.com/xJonathanLEI/starknet-rs", rev = "56177d2" }
+starknet-crypto = { git = "https://github.com/xJonathanLEI/starknet-rs", rev = "56177d2" }
```

Comment:
I'd rather tag it - there seems to be one recent in the repo

Reply:
The recent tag introduced breaking changes:

> The `FieldElement` type has been replaced by `Felt`.

 and the tag before doesn't have support for `v3` transactions.

I would pretty much have to change every file, and I would rather do that migration in a separate PR, not connected to this issue.